src/index.html

```diff
@@ -10,13 +10,106 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Moyo header</title>
+    <title>flexbox</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <a
+          href="#"
+          class="logo"
+        >
+          <img
+            src="images\logo.png"
+            class="header__logo"
+            alt="MOYO Logo"
+          />
+        </a>
+        <nav class="nav">
+          <ul class="nav-list">
+            <li class="nav-item">
+              <a
+                href="#apple"
+                class="nav-link nav-link--apple is-active"
+              >
+                Apple
+              </a>
+            </li>
+
+            <li class="nav-item">
+              <a
+                href="#Samsung"
+                class="nav-link nav-link--samsung"
```

Comment:
Remove this class everywhere
```suggestion
                class="nav-link"
```